### PR TITLE
Accounts

### PR DESCRIPTION
**Pull request by tereshkevich1**

### CODE REVIEW

File: app/src/main/java/com/example/banking/AccountsScreen.kt [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.banking.models.CardState
+import com.example.banking.ui.theme.BankingTheme
+import java.util.Date
+
+@Composable
+fun AccountsScreen() {
+
+    val innerPadding = dimensionResource(id = R.dimen.inner_padding)
```
Comment (Razor0744) [inline app/src/main/java/com/example/banking/AccountsScreen.kt:31]: remove empty line

File: app/src/main/java/com/example/banking/AccountsScreen.kt [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                com.example.banking.models.Transaction("Google", Date(), 1000, CardState.EXECUTED),
+                com.example.banking.models.Transaction("Google", Date(), 1000, CardState.DECLINED),
+                com.example.banking.models.Transaction(
+                    "Google",
+                    Date(),
+                    1000,
+                    CardState.IN_PROGRESS
+                ),
+                com.example.banking.models.Transaction("Google", Date(), 1000, CardState.EXECUTED)
+            )
```
Comment (Razor0744) [inline app/src/main/java/com/example/banking/AccountsScreen.kt:65]: don't do this one, list should be in viewModel

File: app/src/main/java/com/example/banking/CardAccount.kt [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+@Composable
+@Preview(
+    showBackground = true,
+    uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL
+)
+fun CardAccountPreview() {
+    BankingTheme {
+        Surface(
```
Comment (Razor0744) [inline app/src/main/java/com/example/banking/CardAccount.kt:70]: why this fun in this file?

File: app/src/main/java/com/example/banking/AccountsScreen.kt (hunk 1 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/AccountsScreen.kt:31]: fixed

File: app/src/main/java/com/example/banking/CardAccount.kt (hunk 3 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/CardAccount.kt:70]: look at the card with fonts from the theme

File: app/src/main/java/com/example/banking/AccountsScreen.kt (hunk 2 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/AccountsScreen.kt:65]: you said we could hardcode the lists, fixed.

**Pull request merged**